niffler-e-2-e-tests-hw/src/test/java/guru/qa/niffler/data/dao/impl/UserDaoJdbc.java

```diff
@@ -0,0 +1,139 @@
+package guru.qa.niffler.data.dao.impl;
+
+import guru.qa.niffler.config.Config;
+import guru.qa.niffler.data.Databases;
+import guru.qa.niffler.data.dao.UserDao;
+import guru.qa.niffler.data.entity.UserEntity;
+import guru.qa.niffler.model.CurrencyValues;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Optional;
+import java.util.UUID;
+
+public class UserDaoJdbc implements UserDao {
+
+    private final String userdataJdbcUrl = Config.getInstance().userdataJdbcUrl();
+
+    public UserEntity create(UserEntity user) {
+
+        try (Connection connection = Databases.connection(userdataJdbcUrl)) {
+
+            try (PreparedStatement ps = connection.prepareStatement(
+                    "INSERT INTO public.user (username, currency, firstname, surname, photo, photo_small, full_name) " +
```

Comment:
IDEA highlights text "(username, currency, firstname, surname, photo, photo_small, full_name)"

with message:
"Warning:(25, 46) Following columns have no computed/default value and must be listed explicitly: 
account_non_locked, account_non_expired, credentials_non_expired, password, enabled"